content/source/docs/enterprise/private/vault.html.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Customer Managed Vault (Installer Beta)"
+sidebar_current: "docs-enterprise2-private-vault"
+---
+
+# Private Terraform Enterprise Customer Managed Vault
+
+Private Terraform Enterprise can be configured to use an external Vault cluster
+rather than the internal Vault instance. Within PTFE, Vault is used to
+encrypt sensitive information such as variables and states.
```

Comment:
I think it bears clarification on what actually is _stored_ in Vault; the current phrasing may lead a customer to believe vars and states are stored there, not just using encryption-as-a-service.

Reply:
We might store more than just using it for transit eventually so I wanted to customers to treat it like we do.